src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1651,6 +1651,31 @@ GlobeControls.prototype.getScale = function getScale(pitch) {
 };
 
 /**
+ * To convert the projection in meters on the globe of a number of pixels of screen
+ * @param      {number} pixels count pixels to project
+ * @param      {number} pitch Screen pitch, in millimeters
```

Comment:
Could you rename this paramter `pixelPitch` and expand its documentation to explain the default value (`pixel pitch in millimeters (default = 0.28 mm)`